bach/bach/series/series_json.py

```diff
@@ -479,6 +488,36 @@ def _get_slice_partial_expr(self, value: Optional[int], is_start: bool) -> Expre
             array_len = self.get_array_length()
             return Expression.construct(f'({{}} {value})', array_len)
 
+    def _find_in_json_list(self, filtering_slice: Dict[str, str], is_start: bool):
+        """
+        Return expression for the position of the element that contains the filtering values
+
+        Assumes that self._series_object is an array and each element is a json!
+
+        :param filtering_slice: dictionary containing the keys and values to use on filter.
+        :param is_start: whether value is the slice.start (True) or slice.stop (False) value. If true,
+            the first element containing the values is returned, otherwise the last.
+        :return: Expression that gets the position of the element based on the slicing filters.
+        """
+        if not isinstance(filtering_slice, dict):
+            raise TypeError(f'key should be a dict, actual type: {type(filtering_slice)}')
+
+        # this way we can reuse code from the accesor without duplicating
```

Comment:
I was a bit confused about what `element` mean, until I discovered it in the query template below. Maybe this helps for others suffering from local blindness :)

```suggestion
        # this way we can reuse code from the accesor without duplicating.
        # 'element' will filled by iterating over the json array in the query below.
```

Reply:
Done in https://github.com/objectiv/objectiv-analytics/pull/852/commits/10397788d40f0ef62c196d0f1bf6e8e049624847